apps/wallet/src/ui/app/pages/enoki-onboarding/TokensV2.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState } from 'react';
+import { AccountMultiSelect } from '../../components/accounts/AccountMultiSelect';
+import { AccountsList } from '../../components/accounts/AccountsList';
+import { useActiveAddress } from '../../hooks';
+import { useAccounts } from '../../hooks/useAccounts';
+
+export function TokensV2() {
+	const accounts = useAccounts();
+	const address = useActiveAddress();
+	const [selected, setSelected] = useState<string[]>([address!]);
```

Comment:
nit: this might be more readable as to what the state is for 

```suggestion
	const [selectedAddresses, setSelectedAddresses] = useState<SuiAddress[]>([address!]);
```